cms/articles/models.py

```diff
@@ -70,8 +71,8 @@ def minimal_clean(self) -> None:
         super().minimal_clean()
 
     def serve(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> "HttpResponse":
-        # FIXME: redirect to the publications listing for the topic
-        return redirect(self.get_parent().get_url(request=request))
+        # Get the parent topic page
```

Comment:
Comment is not strictly accurate.

Reply:
[9debb4e](https://github.com/ONSdigital/dis-wagtail/pull/387/commits/9debb4effb84627302387d73600b91b94eae9c73)